tests/scattering2d/test_jax_scattering2d.py

```diff
@@ -0,0 +1,285 @@
+import os
+import io
+import jax.numpy as jnp
+import numpy as np
+from kymatio import Scattering2D
+from collections import namedtuple
+import pytest
+
+
+backends = []
+
+from kymatio.scattering2d.backend.jax_backend import backend
+backends.append(backend)
+
+
+class TestPad:
+    @pytest.mark.parametrize('backend', backends)
+    def test_Pad(self, backend):
+        pad = backend.Pad((2, 2, 2, 2), (4, 4), pre_pad=False)
+
+        x = np.random.randn(4, 4) + 1J * np.random.randn(4, 4)
+        x = x[np.newaxis, ...]
```

Comment:
why not just add the axis above when creating `x`

Reply:
Good question.